tests/cmd/git/commands.go

```diff
@@ -33,3 +39,36 @@ func Push(user model.User, keyPath string, app model.App, banner string) {
 	curlCmd = model.Cmd{CommandLineString: fmt.Sprintf(`curl -sL "%s"`, app.URL)}
 	Eventually(cmd.Retry(curlCmd, banner, cmdRetryTimeout)).Should(BeTrue())
 }
+
+// PushWithInterrupt executes a `git push deis master` from the current
+// directory using the provided key, but then halts the progress via SIGINT.
+func PushWithInterrupt(user model.User, keyPath string) {
+	sess := doGitPush(user, keyPath)
+	Eventually(sess.Err).Should(Say("Starting build... but first, coffee!"))
+
+	sess = sess.Interrupt()
```

Comment:
@arschles Yes, good point.  Although gomega's [Eventually](https://onsi.github.io/gomega/#making-asynchronous-assertions) polls the `sess` provided and returns immediately once the supplied string is found (or, if never found, hits the default timeout), it seems anytime 'Eventually' is used on a `sess`, there's no way to actually interrupt/kill/stop that session.  I just noticed when I turned on verbose output, the original `git push deis master` cmd is still ongoing in the background.

Alternatively, if we pull out the `Eventually` check, the cmd is indeed terminated... but then we'd end up needing an even uglier `time.Sleep(duration)` to wait for the cmd to progress a bit before interrupting.  Thoughts?